docs/frontend_utilisation_guide.md

```diff
@@ -93,35 +143,66 @@ document.getElementById("myElementId").appendChild(spsApplication.rootElement);
 
 ### Default Index Implementation
 ```typescript
-import { Config, PixelStreaming } from '@epicgames-ps/lib-pixelstreamingfrontend-ue5.2';
-import { PixelStreamingApplicationStyle } from '@epicgames-ps/lib-pixelstreamingfrontend-ui-ue5.2';
-import { SPSApplication } from "./SPSApplication";
+import {Config, PixelStreaming, SPSApplication, TextParameters, PixelStreamingApplicationStyle} from "@tensorworks/libspsfrontend";
 export const PixelStreamingApplicationStyles = new PixelStreamingApplicationStyle();
```

Comment:
Possibly not something we can change, but it is a little tricky to follow both `spsApplication` and `SPSApplication`

Reply:
This is a coding practice and is fine. `SPSApplication` is the object being imported `spsApplication` is the instance made from that object.